src/components/DropdownMenu.jsx

```diff
@@ -20,11 +20,12 @@ const DowndownMenuTags = () => {
       name="tags"
       id="tags"
       className="select mx-sm text-bold"
+      multiple={false}
       value={input.tags}
       onChange={(e) => setInput({ ...input, tags: [e.target.value] })}
     >
       {tags?.map((tag) => (
-        <option value={tag}>{tag}</option>
+        <option value={tag} key={tag}>{tag}</option>
```

Comment:
Seems vague to have tag handle the key attribute, maybe you could pass some id instead?

Reply:
Actually, I have an array of tags. So, for each tag, giving out an option element under the select. As, it is a value of the tags array. it doesn't require any id. Also, all tags are unique. Hence, passed the tag as a key.